docs/CLI.md

```diff
@@ -74,6 +77,59 @@ $ ./node_modules/.bin/wdio wdio.conf.js
 
 That's it! Now, you can access to the selenium instance via the global variable `browser`.
 
+## Commands
+
+### $ wdio install
+The `install` command allows you to add reporters and services to your WebdriverIO projects via the CLI.
+
+Example:
+
+```bash
+$ wdio install service sauce # installs @wdio/sauce-service
+$ wdio install reporter dot # installs @wdio/dot-reporter
+```
+
+If your project is using `package-lock.json` instead of `yarn.lock`, you can pass a `--npm` flag to make sure the packages are installed via NPM.
+
+```bash
+$ wdio install service sauce --npm
+```
+
+#### List of supported services
+
+```
+    sauce
+    testingbot
+    firefox-profile
+    selenium-standalone
+    devtools
+    applitools
+    browserstack
+    appium
+    chromedriver
+    intercept
+    zafira-listener
+    reportportal
+    docker
+```
+
+#### List of supported reporters
+```
+    dot
+    spec
+    junit
+    allure
+    sumologic
+    concise
+    reportportal
+    video
+    html
+    json
+    mochawesome
+    timeline
+```
+
+
```

Comment:
what about list of supported frameworks?

Reply:
Updated: https://github.com/webdriverio/webdriverio/pull/4010/commits/1fe749a927aa87ff209955ee40f0d5ea46c504c0